packages/decoder/package.json

```diff
@@ -21,10 +21,11 @@
     "build": "tsc",
     "prepare": "yarn build",
     "start": "tsc --watch",
-    "test": "mocha ./test/current/test/* ./test/legacy/test/*"
+    "test": "mocha --exit ./test/current/test/* ./test/legacy/test/* ./test/ens/test/*"
   },
   "types": "dist/index.d.ts",
   "dependencies": {
+    "@ensdomains/ensjs": "^2.1.0",
```

Comment:
We must figure out what impact this has on total package install size!

Reply:
Yeah this is already a transitive dependency via `@truffle/encoder` (I just checked to be certain :P ), so there should be no problem here.

Reply:
Sweet!